### DISCUSSION TITLE
Set dependencies on nested TaskGroup

### DISCUSSION THREAD

**Discussion by antoniocorralsierra** [apache/airflow#34649] [Q&A]
I would like to know if there is any way to set dependencies on child TaskGroup beetwen tasks declarated on parent TaskGroup. Example:

       with TaskGroup(group_id="parent_group") as parent_group:
             op1 = EmptyOperator(task_id="empty_task_1")
             op2 = EmptyOperator(task_id="empty_task_2") 
       
             with TaskGroup(group_id="child_group") as child_group:
                   op1 >> op2

This doesn't work. Appear child_group empty and op1 and op2 sequential on parent_group. I have been searching info but I haven't find nothing. Thanks :)

**Comment by Taragolis** [comment]
Task would assigned into the TaskGroup in which context it created (with block).
If there is no task created into the TaskGroup context than it would be empty.

To be honest it is not clear what you tried to achieve here.

**Reply by antoniocorralsierra** [reply]
Thanks for your answer. I just want reduce the size of my code, nothing of funcionality.
